core/src/test/java/kafka/testkit/KafkaClusterTestKit.java

```diff
@@ -395,9 +395,7 @@ private void formatNodeAndLog(MetaProperties properties, String metadataLogDir,
                 try (PrintStream out = new PrintStream(stream)) {
                     StorageTool.formatCommand(out,
                             JavaConverters.asScalaBuffer(Collections.singletonList(metadataLogDir)).toSeq(),
-                            properties,
-                            MetadataVersion.MINIMUM_BOOTSTRAP_VERSION,
-                            false);
+                            properties, MetadataVersion.MINIMUM_BOOTSTRAP_VERSION, false);
```

Comment:
do we have to do this whitespace change?

Reply:
Nope! Fixed!